nozzle/main.go

```diff
@@ -0,0 +1,103 @@
+package nozzle
+
+import (
+	"context"
+	"crypto/tls"
+	"net/http"
+	"reflect"
+
+	loggregator "code.cloudfoundry.org/go-loggregator"
+	"code.cloudfoundry.org/go-loggregator/rpc/loggregator_v2"
+	noaaerrors "github.com/cloudfoundry/noaa/errors"
+	"github.com/gorilla/websocket"
+	"github.com/wavefronthq/cloud-foundry-nozzle-go/common"
+)
+
+var allSelectors = []*loggregator_v2.Selector{
+	// {
+	// 	Message: &loggregator_v2.Selector_Log{
+	// 		Log: &loggregator_v2.LogSelector{},
+	// 	},
+	// },
+	// {
+	// 	Message: &loggregator_v2.Selector_Counter{
```

Comment:
What are these other selectors? Do they provide any metrics?

Reply:
Those are for events.